packages/devtools_app/lib/src/screens/deep_link_validation/deep_links_model.dart

```diff
@@ -152,7 +159,8 @@ class PathError extends CommonError {
   );
 }
 
-Set<PathError> intentFilterErrors = <PathError>{
+Set<PathError> manifestFileErrors = <PathError>{
+  PathError.missingFlag,
```

Comment:
naming nit: should this be more specific, `missingDeepLinkingFlag`? are there other cases where a "flag" could be missing?

Reply:
Renamed and this is the only case that a flag could be missing